pkg/urbit/daemon/main.c

```diff
@@ -991,6 +993,84 @@ _cw_intr_win(c3_c* han_c)
 }
 #endif
 
+c3_c 
+*_cw_eval_get_input(FILE* fp, size_t size)
+{
+//The size is extended by the input with the value of the provisional
+  c3_c *str;
+  c3_i ch;
+  size_t len = 0;
+
+  str = realloc(NULL, sizeof(*str)*size);//size is start size
+  
+  if(!str)
+    return str;
```

Comment:
You should use our `c3_realloc()` wrapper, which asserts that the returned pointer is non-null. Also, the second parameter can just be `size` (I'm not concerned about the size of `char` changing).